appeals/e2e/cypress/page_objects/caseDetailsPage.js

```diff
@@ -94,9 +94,7 @@ export class CaseDetailsPage extends Page {
 	}
 
 	uploadOneDocument() {
-		cy.get('.pins-file-upload__dropzone').selectFile('cypress/fixtures/sample-doc.pdf', {
-			action: 'drag-drop'
-		});
+		cy.get('#upload-file-1').selectFile('cypress/fixtures/sample-doc.pdf', { force: true });
```

Comment:
As per the pattern used in the rest of the file we shouldn't have the `#upload-file-1` locator within the method here, it should be moved to a `selectors` object at the top of the file, so

```
	selectors = {
		uploadFile: '#upload-file-1',
        }
```

and then referenced in the elements objects

```
	elements = {
		uploadFile: () => cy.get(this.selectors.uploadFile)
        }
```